### PR TITLE
Update HTML/CSS docs

### LINKED ISSUE #1435: [Docs Feedback]: HTML/CSS docs outdated
### Page URL

https://www.gitpod.io/docs/languages/html

### User feedback

Reported on Discord.

There is no `Open Preview to the Side` icon. The screenshot uses Theia.

### PR DESCRIPTION
**Pull request by pawlean**
## Description

This updates the HTML/CSS docs as requested by someone from the community.

## Related Issue(s)

Fixes #1435 

## How to test

In Gitpod! 🧡

## Release Notes

```release-note
```

## Documentation

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1437"></a>

### CODE REVIEW

File: gitpod/docs/languages/html.md [hunk 1]
```diff
@@ -31,15 +31,23 @@ Gitpod comes with [emmet](https://emmet.io/) support right out of the box giving
...
-![Live Preview Demo](../../../static/images/docs/live_preview.png)
-With Gitpod you can open a preview for Markdown and HTML files while you are coding.
+![Live Preview Demo](../../../static/images/docs/html-preview.png)
+With Gitpod you can open a preview for HTML files while you are coding. You can do this by opening a web server `python -m http.server 8000`.
 
-To open the preview for a file, click the `Open Preview to the Side` icon in the upper right-hand corner:
+You can also automate this in your [`.gitpod.yml`](https://www.gitpod.io/docs/config-gitpod-file) file, so that every time you start a new workspace your preview is ready to go. For example:
 
-![preview icon](../../../static/images/docs/preview.png)
+```
```
Comment (Cahllagerfeld) [inline gitpod/docs/languages/html.md:39]:
```suggestion
```yaml
```
Adding `yaml` there provdes syntax highlighting for yaml

**Comment by Cahllagerfeld** [comment]
grafik

I don't know if the above screenshot shows Theia as well, but it definitely uses the old Gitpod-Logo, maybe we should tackle this as well 🤔

**Comment by pawlean** [comment]
> grafik
> 
> I don't know if the above screenshot shows Theia as well, but it definitely uses the old Gitpod-Logo, maybe we should tackle this as well 🤔

You're right there -- I've updated the screenshot to show Emmet in VS Code (old screenshot was Theia)

**Review by Cahllagerfeld — APPROVED** [review]
Looks good to me 🚀

**Pull request merged**